app/actions/EventActions.js

```diff
@@ -203,7 +203,7 @@ export function adminRegister(
     method: 'POST',
     body: {
       user: userId,
-      pool: poolId,
+      pool: poolId ? poolId : undefined,
```

Comment:
Instead of this ternary, maybe set the flowtype to `poolId?: number` instead and not pass poolId when calling the function? That way it will implicitly be undefined.

Reply:
You may handle the value in the component here instead if you want to change it https://github.com/webkom/lego-webapp/blob/master/app/routes/events/components/EventAdministrate/index.js#L80